app/controllers/api/v0/swagger/models/highlight.rb

```diff
@@ -114,6 +116,19 @@ module Api::V0::Swagger::Models::Highlight
     end
   end
 
+  add_properties(:HighlightUpdate) do
```

Comment:
can we keep the same pattern for others?  :UpdateHighlight (symmetrical to NewHighlight)

Reply:
I had named it like this b/c it isn't a full Highlight, it is just an update to a highlight... but I think I could get to the other way of thinking that the name is "VerbHighlight"